### DISCUSSION TITLE
Bug: Memory Leak

### DISCUSSION THREAD

**Discussion by Tsukasuke** [anapnoe/sd-webui-ux#12] [Q&A] [answered]
[**OS]:**   _Windows 10_
[**UI:**]   _Forge (latest commit f4a6a08)_
[**Browser:**]   _DuckDuckGo Preview_
[**GPU:**]   _nVidia 4080S_
[**CPU:**]   _Intel i7-14700K_

[**Issue:**]   _Memory Usage slowly climbs until the browser crashes._
[**Evidence:**]   _Observed in Task Manager under DuckDuckGo Preview task in Processes tab> "Microsoft Edge WebView2".  Console Log repeatedly returns "Attempted to select a non-interactive or hidden tab." approx. 20-25x / sec._
[**Steps to Recreate:**]   _Open Settings and click any option._
[**Resolution Attempts:**]  
-  _Swapped to a different browser "Microsoft Edge Version 131.0.2903.51 (Official build) (64-bit)", issue persists._ 
- _Removed the UI/UX extension, issue ceases._
- _Removed all extensions, reintroducing them one by one._  **Leak found when using [SD-WebUI-LivePortrait](https://github.com/dimitribarbot/sd-webui-live-portrait)** (latest commit **f379b89**) _along with this extension. LivePortrait extension by itself causes no issues._

Like many that pass through with this stuff, I'm a big fan with little technical knowledge, but can supply any other information needed.  Thank you so much for your contributions, I love this UI.

**Comment by anapnoe** [comment, marked as answer]
Hi thank you, can you test, on Chrome, and Mozilla browsers? I can see the message "Attempted to select a non-interactive or hidden tab." but this message comes out even when webui-ux extension is disabled and no memory climbs up in my system so I believe it may be some conflict with another extension, you can try to disable all extensions and enable them one by one to find what causes the issue

**Reply by Tsukasuke** [reply]
I greatly appreciate your time.  I haven't tested with the other browsers yet, but did find that the leak only occurred when I had the LivePortrait extension installed along with this one.
